syntax = "proto3";

package AdaptiveModels;

service WeldBeadModel {
  rpc inference (InputVector) returns (OutputVector) {}
}

message InputVector {
  repeated double data = 210 [packed=true];
}

message OutputVector {
  repeated double data = 216 [packed=true];
}
